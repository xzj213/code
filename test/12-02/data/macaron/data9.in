10
2087 3863 2087 3863 3863 0 0 0 0 2087
3863 3863 2087 3863 3863 3863 2087 3863 19 3863
0 3863 1010 0 3863 3863 2087 2087 3863 3863
0 2087 2087 0 2087 0 3863 0 1317 2087
2087 2087 0 0 0 0 3863 2087 3863 0
2087 0 0 0 3863 2087 3863 0 3863 0
3863 0 0 3863 0 3863 3863 3863 0 3863
0 0 0 2087 3863 3863 0 2087 0 3863
2087 2087 0 3863 3863 0 0 0 0 2087
3863 2087 2087 0 0 0 0 0 2087 2087
