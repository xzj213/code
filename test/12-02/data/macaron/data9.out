0 19 487 533 993 1010 1218 1233 1317 1751 1827 1840 2087 2100 2496 2610 3014 3029 3301 3318 3330 3824 3844 3863 
